src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/SSHUserPrivateKeyBinding/help.html

```diff
@@ -0,0 +1,9 @@
+<div>
+    Copies the SSH key file given in the credentials to a temporary location, then sets a variable to that location.
+    (The file is deleted when the build completes.) Also sets variables for the SSH key's username and passphrase.
```

Comment:
Nice to create `help-passphraseVariable.html` giving specifics, such as that two parameters are optional. Can also add `doCheck*` methods to the descriptor guiding the user to fill in `keyFileVariable` but allowing the others to remain blank.